### PR TITLE
All the terms

### PR DESCRIPTION
**Pull request by lorikrell**
## Scope of this PR

SUPER MEGA!

## Purpose of this PR (required)

Add alot of terms and stuff.
All terms have correct metadata.
Added some news one, including PWA. 
Merge away when ya can. Will continue terms on Friday morning.

### CODE REVIEW

File: src/_converted/entity.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-  tags: 
+  tags:
     - "magento-software"
   synonyms: []
-  relatedTerms: 
+  relatedTerms:
     - "attribute"
     - "cart rule"
     - "catalog rule"
+  relatedTerms: []
```
Comment (jcalcaben) [inline src/_converted/entity.md]:
```suggestion
```
Duplicate entry. Causing error on build.

**Review by jcalcaben — CHANGES_REQUESTED** [review]
Comment added inline

File: src/_converted/entity.md (hunk 1 above)
Comment (lorikrell) [reply src/_converted/entity.md]: Fixed!

**Comment by lorikrell** [comment]
Pushed update!

**Review by jcalcaben — APPROVED** [review]

**Pull request merged**